generator/generator_base.rb

```diff
@@ -76,7 +80,7 @@ def module_yml_out_path
       end
 
       def sequence_out_path
-        File.expand_path("#{module_yml_out_path}/#{path}")
+        File.expand_path("#{module_yml_out_path}/#{prefix + '_' if prefix.present?}#{path}")
```

Comment:
I think it would be nice to have a method like this so that this logic doesn't have to be repeated:
```
def prefix_string
  prefix.present? ? prefix + '_' : ''
end
```

Reply:
This is fixed. I also changed it to behave like before when none of the new params are specified.

Reply:
Could you update the PR description so that its instructions match the current implementation? If I want to remember how to use this functionality I'm going to find this PR and look at those instructions.